docs/MIL/coremltools.converters.mil.ops.rst

```diff
@@ -170,12 +171,13 @@ The list of operators supported by MIL.
   .. autoclass:: tile
   .. autoclass:: argsort
   .. autoclass:: topk
-  .. autoclass:: flatten
+  .. autoclass:: flatten2d
   .. autoclass:: shape
   .. autoclass:: concat
   .. autoclass:: split
   .. autoclass:: stack
-  .. autoclass:: addn
+  .. autoclass:: identity
+  .. autoclass:: isfinite
```

Comment:
Is this PyMIL op list? I think `isfinite` is no longer a PyMIL op (it's using composite ops now), but maybe change will show up in 4.0b4.

Reply:
This is really just a sync with what's in the GitHub repo right now.  We'll need to update as newer releases come.